plugins/modules/ostree_distribution.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# copyright (c) 2019, Matthias Dellweg
```

Comment:
I don't think, you should put my copyright there...
Also can you add yourself to the authors in galayx.yml?

Reply:
Copy/paste error